app/serializers/serialize_submission_test_run.rb

```diff
@@ -37,7 +37,10 @@ def message
   def output
     return if test_run.output.blank?
 
-    Ansi::To::Html.new(test_run.output).to_html
+    # The ansi-to-html library does not support unicode escape sequence
+    # See https://github.com/rburns/ansi-to-html/issues/48
+    sanitized_output = test_run.output.gsub(/\e\[K/, '')
```

Comment:
Does this need to be a regexp? Could you use a string (and/or `tr`) instead?

Reply:
I've used a string now